data/campaigns/World_Conquest/lua/game_mechanics/unittypedata.lua

```diff
@@ -153,93 +180,113 @@ type_infos = {
 		} }
 	},
 	["Dwarvish Fighter"] = { 
+		-- po: a Dwarvish Fighter joins a player's side
 		founddialogue=_"Having trouble, eh? Never worry, lads, we’ll sort ’em out soon enough!",
 		image="units/dwarves/fighter.png",
 		name="Dwarvish Fighter",
 	},
 	["Thief"] = { 
+		-- po: a Thief joins a player's side. This one's British slang for "you've caught me stealing, let me work for you instead", with extra words that are just parts of the idiom.
 		founddialogue=_"You’ve got me, guv, it’s a fair cop! Just lemme work for you instead. You won’t regret it, guv, I promise!",
 		image="units/human-outlaws/thief.png",
 		name="Thief",
 	},
 	["Dwarvish Thunderer"] = { 
+		-- po: a Dwarvish Thunderer joins a player's side
 		founddialogue=_"Listen up, ye primitive screwheads! This... is me BOOM STICK. Lemme show you what this baby can do!",
 		image="units/dwarves/thunderer/thunderer.png",
 		name="Dwarvish Thunderer",
 	},
 	["Poacher"] = { 
+		-- po: a Poacher joins a player's side
 		founddialogue=_"What, you want my help? A guy like me? Huh, that’s rich. Oh well... let’s give it a shot, eh?",
 		image="units/human-outlaws/poacher.png",
 		name="Poacher",
 	},
 	["Dwarvish Guardsman"] = { 
+		-- po: a Dwarvish Guardsman joins a player's side
 		founddialogue=_"A soldier is no good without something to fight for. Let me fight for you!",
 		image="units/dwarves/guard.png",
 		name="Dwarvish Guardsman",
 	},
 	["Footpad"] = { 
+		-- po: a Footpad joins a player's side. The first half is "please don’t attack me", the second is "you seem to be in a difficult situation too, I’m good at talking or fighting my way out".
 		founddialogue=_"Hey, hey, easy there! I done nothin’ to hurt you. We’re all friends here, right? Looks like you might be in a tight spot, but don’t worry. No one’s better at getting out of tight spots than me, boss!",
 		image="units/human-outlaws/footpad.png",
 		name="Footpad",
 	},
 	["Dwarvish Ulfserker"] = { 
+		-- po: an Ulfserker joins a player's side
 		founddialogue=_"Chin up, lads. Today is a good day to die!",
 		image="units/dwarves/ulfserker.png",
 		name="Dwarvish Ulfserker",
 		alt_reply = { { 
 			race="dwarf",
+			-- po: the speaker is another dwarf, after an Ulf joins their side
 			reply=_"Brave words. Welcome to The Fight Club...",
 		}, {
 			race="human",
+			-- po: the speaker is a human, after an Ulf joins their side. TODO: how does this not result in an immediate fight?
 			reply=_"Brave words. Welcome little big man.",
```

Comment:
Can't be changed for 1.16, but making a height joke to an Ulf who you've just met seems a foolish act.

Reply:
Sometimes, the role of a PR is to catch what-was-I-thinking commits like this. Thanks, I'll remove the TODOs and make an issue for them.

Reply:
If it's meant to be a reference to [a film](https://en.wikipedia.org/wiki/Little_Big_Man_(film)) or [a real Native American Chief](https://en.wikipedia.org/wiki/Little_Big_Man), then I guess "Brave words" is also a reference that I missed.

However, a PO hint should either say something about the references, or suggest an alternative that could be translated without the references.

Reply:
The comments have been updated, and the TODO removed (although I still think the line is likely to change in 1.17).

Github isn't marking this as `Outdated`, even though it is outdated, because this comment thread is on the unchanged line rather than the `po` line.

Reply:
I agree with soliton here. IMO its fine but I dont have any strong feelings about that. 

At least it sounds like a good way to make ulf follow you. :P